v2/event/event_marshal_test.go

```diff
@@ -66,7 +66,7 @@ func TestMarshal(t *testing.T) {
 				"exurl":    source,
 				"extime":   &now,
 			},
-			want: toBytes(map[string]interface{}{
```

Comment:
In this file we need to add a test for each of the other encodings the sdk supports